2-classes-db/-- TASK 1 transaction between 30000-to-31000.pgsql

```diff
@@ -0,0 +1,4 @@
+-- Show the date, transaction_no, description and 
+-- amount for those transactions whose amount is between £30,000 and £31,000
+
+SELECT date,transaction_no,description, amount FROM spends WHERE 30000 < amount AND amount < 31000;
```

Comment:
There's a little typo here. Can you see it?

Reply:
I thing yes, it is about the WHERE key word instead of BETWEEN. If not that I have no Idea what that Typo is😀.